themes/round-hp/assets/js/components/pokemon.js

```diff
@@ -0,0 +1,112 @@
+Vue.component("Pokemon", {
+  template: `
+    <div 
+      :class="{'pkmn': true, 'pkmn--fainted': isFainted, 'pkmn--empty': isEmptySlot}" 
+      :style="{'background': getBackgroundColor }"
+    >
+      <div v-if="typeof pokemon == 'object'" class="pkmn__wrapper">
+        <svg viewBox="0 0 36 36" class="pkmn__hp">
+          <path
+            d="M18 2.0845
+              a 15.9155 15.9155 0 0 1 0 31.831
+              a 15.9155 15.9155 0 0 1 0 -31.831"
+            fill="none"
+            :stroke="getHPColor"
+            stroke-width="1.5"
+            :stroke-dasharray="getHP + ', 100'"
+          />
+        </svg>
+        <img v-if="pokemon.isEgg" class="pkmn__sprite" :src="pokemon.img" style="max-height: 80px;" />
+        <img v-else class="pkmn__sprite" :src="pokemon.img" />
+        <div class="pkmn__details">
+          <div class="details__name">
+            {{pokemon.nickname.toLowerCase()}}
```

Comment:
What about if there's no nickname?

Reply:
I believe that PokeLink provides the species name as the nickname if it's missing. Regardless, I've updated the code, as I meant to use the `nickname` computed value, not the one coming from the settings